client/tests/integration/components/land-use-action-test.js

```diff
@@ -0,0 +1,122 @@
+
+    assert.dom(this.element).includesText('How many Change in CityMap actions?');
+
+    assert.ok(!this.package.pasForm.dcpPfchangeincitymap);
+
+    await fillIn('[data-test-input="dcpPfchangeincitymap"]', 4);
+
+    await triggerEvent('[data-test-input="dcpPfchangeincitymap"]', 'keyup');
+
+    assert.equal(this.package.pasForm.dcpPfchangeincitymap, 4);
+
+    // Check that we can add "Zoning Special Permit" and answer extra questions
+    await selectChoose('[data-test-land-use-action-picker]', 'Zoning Special Permit');
+
+    assert.dom(this.element).includesText('How many Zoning Special Permit actions?');
+    assert.dom(this.element).includesText('Where in the Zoning Resolution can this action be found?');
+    assert.dom(this.element).includesText('Which sections of the Zoning Resolution does this modify?');
+
+    assert.ok(!this.package.pasForm.dcpPfzoningspecialpermit);
+    assert.ok(!this.package.pasForm.dcpZoningspecialpermitpursuantto);
+    assert.ok(!this.package.pasForm.dcpZoningspecialpermittomodify);
+
+    await fillIn('[data-test-input="dcpPfzoningspecialpermit"]', 6);
+    await triggerEvent('[data-test-input="dcpPfzoningspecialpermit"]', 'keyup');
+
+    assert.equal(this.package.pasForm.dcpPfzoningspecialpermit, 6);
+
+    await fillIn('[data-test-input="dcpZoningspecialpermitpursuantto"]', 'Section 5B');
+    await triggerEvent('[data-test-input="dcpZoningspecialpermitpursuantto"]', 'keyup');
+
+    assert.equal(this.package.pasForm.dcpZoningspecialpermitpursuantto, 'Section 5B');
+
+    await fillIn('[data-test-input="dcpZoningspecialpermittomodify"]', 'Permit 7A');
+    await triggerEvent('[data-test-input="dcpZoningspecialpermittomodify"]', 'keyup');
+
+    assert.equal(this.package.pasForm.dcpZoningspecialpermittomodify, 'Permit 7A');
+  });
+
+  test('User can delete an action', async function(assert) {
+    const projectPackage = this.server.create('package');
+
+    this.package = await this.owner.lookup('service:store').findRecord('package', projectPackage.id, { include: 'pas-form' });
+
+    // Template block usage:
+    await render(hbs`
+      <LandUseAction @pasForm={{this.package.pasForm}}>
+      </LandUseAction>
+    `);
+
+    // Check that we can delete "Change in CityMap"
+    await selectChoose('[data-test-land-use-action-picker]', 'Change in CityMap');
+
+    assert.dom(this.element).includesText('How many Change in CityMap actions?');
+
+    assert.ok(!this.package.pasForm.dcpPfchangeincitymap);
+
+    await fillIn('[data-test-input="dcpPfchangeincitymap"]', 4);
+
+    await triggerEvent('[data-test-input="dcpPfchangeincitymap"]', 'keyup');
+
+    assert.equal(this.package.pasForm.dcpPfchangeincitymap, 4);
+
+    await click('[data-test-delete-button="Change in CityMap"]');
+
+    assert.dom(this.element).doesNotIncludeText('How many Change in CityMap actions?');
+
+    assert.equal(this.package.pasForm.dcpPfchangeincitymap, 0);
```

Comment:
Great, this seems like this should be the end of the test. What is the rest for? Maybe there's some other behavior you want to check? Seems like it's another part of the form. I think if that's the case, it should be a separate `test` block.

On the other hand, if you are throwing in these extra assertions just to make _extra_ sure that your component works, I'd say you should give yourself some credit: the test you have already is concise and tests for the behavior you want.